frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -61,12 +63,12 @@ const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight,
     () => [
       {
         href: '',
-        name: 'Cluster',
+        name: t('dashboard~Cluster'),
         component: ClusterDashboard,
       },
       ...pluginPages,
     ],
-    [pluginPages],
+    [pluginPages, t],
```

Comment:
Can you confirm that `t` does not change causing this to run again and again?

Reply:
@spadgett validated the HorizontalNav that uses allPages only renders once.